### PR TITLE
Path of Repositories ends with namespace and name

### PR DESCRIPTION
**Pull request by baudronp**

### CODE REVIEW

File: NGitLab.Mock/Repository.cs [hunk 1]
```diff
@@ -67,9 +68,11 @@ private LibGit2Sharp.Repository GetGitRepository()
                     if (_directory == null)
                     {
                         var directory = TemporaryDirectory.Create();
+                        var repositoryDirectory = Path.Combine(directory.FullPath, Project.PathWithNamespace.Replace('/', Path.DirectorySeparatorChar));
```
Comment (meziantou) [inline NGitLab.Mock/Repository.cs:73]: Can you add a comment to explain why having the namespace in the path is important

File: global.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
     "sdk": {
         "version": "6.0.202",
-        "rollForward": "latestFeature"
+        "rollForward": "major"
```
Comment (louis-z) [inline global.json:4]: Not sure we want this.

File: NGitLab.Mock.Tests/ProjectsMockTests.cs [hunk 3]
```diff
@@ -35,6 +35,23 @@ public void Test_project_can_be_cloned_by_default()
...
+                .WithUser("Test", isDefault: true)
+                .WithProject("Test", @namespace: "testgroup", addDefaultUserAsMaintainer: true)
+                .BuildServer();
+
+            var client = server.CreateClient();
+            var project = client.Projects["testgroup/Test"];
+
+            Assert.IsNotNull(project);
+            StringAssert.EndsWith("testgroup\\test", project.SshUrl);
+            StringAssert.EndsWith("testgroup\\test", project.HttpUrl);
```
Comment (louis-z) [inline NGitLab.Mock.Tests/ProjectsMockTests.cs]: Should be slashes, not backslashes, right?

File: global.json (hunk 2 above)
Comment (meziantou) [reply global.json:4]: Most people don't have a compatible SDK on their machine. Using `major`, you can open and build the solution correctly.

File: NGitLab.Mock.Tests/ProjectsMockTests.cs (hunk 3 above)
Comment (baudronp) [reply NGitLab.Mock.Tests/ProjectsMockTests.cs]: They are "files://" uri with '\' in them.

Comment (baudronp) [reply NGitLab.Mock.Tests/ProjectsMockTests.cs]: in fact this seems to be platform dependant... I fix it shortly

**Pull request merged**